### PR TITLE
Releasing v0.24.4 of fluxcd-kustomize-controller

### PR DESCRIPTION
**Pull request by chaudharyalka**
## What this PR does / why we need it

Releasing v0.24.4 of fluxcd-kustomize-controller
Adding latest version v0.24.4 of fluxcd-kustomize-controller compatible with source-controller latest version

## Describe testing done for PR

1. Create a kind cluster and attach it to TMC.
2. Submit the metadata.yaml and 0.24.4/package.yaml files
3. Install the package using the tanzu cli
tanzu package install fluxcd-kustomize-controller -p kustomize-controller.fluxcd.community.tanzu.vmware.com -v 0.24.4
/ Installing package 'kustomize-controller.fluxcd.community.tanzu.vmware.com'
| Getting package metadata for 'kustomize-controller.fluxcd.community.tanzu.vmware.com'
| Creating service account 'fluxcd-kustomize-controller-default-sa'
| Creating cluster admin role 'fluxcd-kustomize-controller-default-cluster-role'
| Creating cluster role binding 'fluxcd-kustomize-controller-default-cluster-rolebinding'
| Creating package resource
/ Waiting for 'PackageInstall' reconciliation for 'fluxcd-kustomize-controller'
\ 'PackageInstall' resource install status: Reconciling

Added installed package 'fluxcd-kustomize-controller'

Added installed package 'fluxcd-kustomize-controller'

### CODE REVIEW

File: addons/packages/fluxcd-kustomize-controller/0.24.4/package.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  valuesSchema:
+    openAPIv3:
+      properties:
+        namespace:
+          type: string
+          description: Deployment and service namespace
+          default: kustomize-system
+        limits_cpu:
+          type: string
+          description: Set cpu usuage limit
```
Comment (seemiller) [inline addons/packages/fluxcd-kustomize-controller/0.24.4/package.yaml:34]:
```suggestion
          description: Set cpu usage limit
```

Comment (seemiller) [inline addons/packages/fluxcd-kustomize-controller/0.24.4/package.yaml:34]:
```suggestion
          description: Set cpu usage limit
```
```suggestion
          description: Set cpu usuage limit
```

File: addons/packages/fluxcd-kustomize-controller/0.24.4/package.yaml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+          default: 1000m
+        limits_memory:
+          type: string
+          description: Set memory usuage limit
```
Comment (seemiller) [inline addons/packages/fluxcd-kustomize-controller/0.24.4/package.yaml:38]:
```suggestion
          description: Set memory usage limit
```

Comment (seemiller) [inline addons/packages/fluxcd-kustomize-controller/0.24.4/package.yaml:38]:
```suggestion
          description: Set memory usage limit
```

**Review by seemiller — CHANGES_REQUESTED** [review]

File: addons/packages/fluxcd-kustomize-controller/0.24.4/package.yaml (hunk 1 above)
Comment (chaudharyalka) [reply addons/packages/fluxcd-kustomize-controller/0.24.4/package.yaml]: Done

Comment (chaudharyalka) [reply addons/packages/fluxcd-kustomize-controller/0.24.4/package.yaml]: Done

File: addons/packages/fluxcd-kustomize-controller/0.24.4/package.yaml (hunk 2 above)
Comment (chaudharyalka) [reply addons/packages/fluxcd-kustomize-controller/0.24.4/package.yaml]: Done

Comment (chaudharyalka) [reply addons/packages/fluxcd-kustomize-controller/0.24.4/package.yaml]: Done

**Review by seemiller — APPROVED** [review]

**Pull request merged**